internal/logger/logger.go

```diff
@@ -1 +1,22 @@
 package logger
```

Comment:
We may need to get rid of this file and use the `logger.go` in the current `master` branch.

Reply:
sure..I'll do that tonight